aries-site/src/examples/templates/dashboards/DashboardExample.js

```diff
@@ -1,6 +1,212 @@
-import React from 'react';
-import { DashboardTiles } from '.';
-import { AppHeaderExample, PageHeaderExample, ScreenContainer } from '..';
+import React, { useContext } from 'react';
+import { AppIdentity, Identifier, Tiles, Tile } from 'aries-core';
```

Comment:
I think we are steering away from using any 'aries-core' components since the library hasn't been published yet. We might need to create components here call "AppIdentity", etc. -- but would be good to get clarification here on approach

Reply:
yeah I wasnt to sure about that as well. I think we need more clarification on what we should show or make the same type of component with grommet